src/App.test.js

```diff
@@ -1,8 +1,54 @@
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import userEvent from '@testing-library/user-event';
+import App from './app';
 
-test('renders learn react link', () => {
+window.scroll = jest.fn();
+
+test('renders the Home page', () => {
+  render(<App />);
+  expect(
+    screen.getAllByText(/Welcome to the Codecademy Discord Community/i)[0]
+  ).toBeInTheDocument();
+});
+
+test('navigates to and renders the Server Staff page', async () => {
+  render(<App />);
+  const user = userEvent.setup();
+  await user.click(screen.getByTestId('server-staff-link'));
```

Comment:
Test IDs should really be thought of as last resorts. 
Rather than alter our code so that we can test it, let's write our tests by using elements that the user can actually see.

For example, this test could be rewritten as 
```js
it('navigates to and renders the Server Staff page', async () => {
  render(<App />);
  const user = userEvent.setup();
  const serverStaffLink = screen.getAllByText('Server Staff')[0];
  await user.click(serverStaffLink)
  expect(screen.getAllByText(/Admins/)[0]).toBeInTheDocument();
});
```

In that way, we are testing what happens when the user clicks on the first "Server Staff" text in the page, and we don't need to alter our code to do it.